charger/warp/types.go

```diff
@@ -86,7 +86,7 @@ type EmConfig struct {
 	PhaseSwitchingMode int  `json:"phase_switching_mode"`
 }
 
-//go:generate enumer -type ExternalControl -trimprefix ExternalControl -transform whitespace
+//go:generate go run github.com/dmarkham/enumer@v1.5.8 -type ExternalControl -trimprefix ExternalControl -transform whitespace
```

Comment:
`make install` and `tools.go` could be removed with this change

Reply:
We should keep this as-is. Its ungelated to this PR anyway 🤗